src/main/blackbox/blackbox_io.c

```diff
@@ -102,9 +124,32 @@ void blackboxWrite(uint8_t value)
 #endif
     case BLACKBOX_DEVICE_SERIAL:
     default:
+#ifdef DEBUG_BB_OUTPUT
```

Comment:
@jflyper:

> I'm wondering if the code should always be compiled in, or enabled only when this code is necessary.

Interesting one - I know this is somewhat chicken and egg, but I think it makes sense to have this always built: Even if users cannot use this to generate logs with this debug switch to find out why logging does not work (:grinning:), they can still be directed to generate a plot in configurator, and post a screenshot, giving us a better chance at diagnosing than without any data.